utils/post_processing/bfiq_to_rawacf.py

```diff
@@ -0,0 +1,157 @@
+import deepdish as dd
+import sys
+import subprocess as sp
+import os
+from datetime import datetime as dt
+import warnings
+
+def bfiq_to_rawacf_postprocessing(bfiq_filepath):
+	"""
+	Processes the data from a bfiq.hdf5 file and creates auto and cross correlations from the samples.
+	This data is formatted and written to mimic borealis rawacf.hdf5 files.
+	"""
+
+	def correlate_samples(ts_dict):
+		"""
+		Builds the autocorrelation and cross-correlation matrices for the beamformed data
+		contained in one timestamped dictionary
+		:param ts_dict: A timestamped dictionary from a formated bfiq.hdf5 file
+		"""
+		data_buff = ts_dict["data"]
+		num_slices = ts_dict["num_slices"]
+		num_ant_arrays = ts_dict["data_dimensions"][0]
+		num_sequences = ts_dict["data_dimensions"][1]
+		num_beams = ts_dict["data_dimensions"][2]
+		num_samples = ts_dict["data_dimensions"][3]
+		dims = ts_dict["data_dimensions"]
+
+		lags = ts_dict["lags"]
+		num_lags = np.shape(ts_dict["lags"])[0]
+		num_ranges = ts_dict["num_ranges"]
+		num_slices = ts_dict["num_slices"]
+		
+		data_mat = data_buff.reshape(dims)
+
+		# Get data from each antenna array
+		main_data = data_mat[0][:][:][:]
+		intf_data = data_mat[1][:][:][:]
+
+		# Preallocate arrays for correlation results
+		main_corrs = np.zeros((num_sequences, num_beams, num_samples, num_samples), dtype=np.complex64)
+		intf_corrs = np.zeros((num_sequences, num_beams, num_samples, num_samples), dtype=np.complex64)
+		cross_corrs = np.zeros((num_sequences, num_beams, num_samples, num_samples), dtype=np.complex64)
+
+		# Preallocate arrays for results of range-lag selection
+		out_main = np.zeros((num_sequences, num_beams, num_ranges, num_lags), dtype=np.complex64)
+		out_intf = np.zeros((num_sequences, num_beams, num_ranges, num_lags), dtype=np.complex64)
+		out_cross = np.zeros((num_sequences, num_beams, num_ranges, num_lags), dtype=np.complex64)
+
+		# Perform autocorrelations of each array, and cross
+		# correlation between arrays
+		for seq in range(num_sequences):
+			for beam in range(num_beams):
+				main_samps = main_data[seq, beam]
+				intf_samps = intf_data[seq, beam]
+
+				main_corrs[seq, beam] = np.outer(main_samps.conjugate(), main_samps)
+				intf_corrs[seq, beam] = np.outer(intf_samps.conjugate(), intf_samps)
+				cross_corrs[seq, beam] = np.outer(main_samps.conjugate(), intf_samps)
+
+				beam_offset = num_beams * num_ranges * num_lags
+				first_range_offset = int(ts_dict["first_range"] / ts_dict["range_sep"])
+
+				# Select out the lags for each range gate
+				main_small = np.zeros((num_ranges, num_lags,), dtype=np.complex64)
+				intf_small = np.zeros((num_ranges, num_lags,), dtype=np.complex64)
+				cross_small = np.zeros((num_ranges, num_lags,), dtype=np.complex64)
+
+				for rng in range(num_ranges):
+					for lag in range(num_lags):
+						# tau spacing in us, sample rate in hz
+						tau_in_samples = np.ceil(ts_dict["tau_spacing"] * 1e-6 * 
+													ts_dict["rx_sample_rate"])
```

Comment:
Regarding this line continuation: https://www.python.org/dev/peps/pep-0008/#maximum-line-length
PEP8 is a good styleguide to follow. If you use pycharm (which I really like for python development) it can tell you if you're not adhering to pep8 (among other static checks like typos, incorrect python grammar, etc)